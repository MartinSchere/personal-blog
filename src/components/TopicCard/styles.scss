.image-wrapper {
  position: relative;
  display: flex;
  align-items: center;
  height: 2.5rem;
  justify-content: center;

  min-width: 150px;
  word-break: keep-all;

  @include respond-to("large") {
    width: 235px;
  }

  .topic-name {
    color: white;
    z-index: 3;
    padding: 0 1rem;
    font-weight: 400;
    font-size: 1.15rem;
  }
  .topic-image {
    width: 100%;
    position: absolute;
    opacity: 0.9;
    height: 2.5rem;
    border-radius: 5px;
  }
}
