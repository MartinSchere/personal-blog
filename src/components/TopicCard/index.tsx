import React from 'react';
import "./styles.scss";

type TopicCardProps = {

}

const TopicCard = (props:TopicCardProps) => {
return (
<div>
  TopicCard
</div>
);
};

export default TopicCard;