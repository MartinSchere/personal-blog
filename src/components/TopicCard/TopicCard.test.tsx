import React from 'react';
import TopicCard from './index';
import renderer from 'react-test-renderer';

describe('TopicCard', () => {
it('renders without error', () => {
const tree = renderer
.create(<TopicCard></TopicCard>)
.toJSON();
expect(tree).toMatchSnapshot();
});
});