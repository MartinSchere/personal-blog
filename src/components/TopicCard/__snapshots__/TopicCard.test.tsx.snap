// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopicCard renders without error 1`] = `
<div>
  TopicCard
</div>
`;
